documentation/docs/25-build-and-deploy/70-adapter-cloudflare-workers.md

```diff
@@ -36,6 +37,37 @@ export default {
 
 Path to your custom `wrangler.toml` or `wrangler.json` config file.
 
+### handlers
+
+Path to a file with additional [handlers](https://developers.cloudflare.com/workers/runtime-apis/handlers/) and (optionally) [Durable Objects](https://developers.cloudflare.com/durable-objects/) to be exported from the file the adapter generates. This allows you to, for example, include handlers for scheduled or queue triggers alongside the fetch handler your SvelteKit app.
+
+The handlers file should export a default object with any additional handlers, and any Durable Objects as named exports. Example below:
+
+```js
```

Comment:
The docs are failing to deploy. We might need this line to ignore the error where "cloudflare:workers" can't be resolved in the code block.
```suggestion
```js
// @errors: 2307
```